PublicationsRouter/api/v3/Retrieve.md

```diff
@@ -112,35 +112,21 @@ If the notification is found and has been routed, you will receive a 200 (OK) an
 
 See the [Outgoing Notification](./OutgoingNotification.md#outgoing-notification) data model for more info.
 
-Some notifications may contain one or more **links** elements.  In this event, this means that there is binary content associated with the notification available for download.  Each of the links could be one of two kinds:
+Some notifications may contain one or more **links** elements. This means that there is binary content associated with the notification available for download.  The link will be packaged binary content held by PubRouter on behalf of the publisher.
 
-1. Packaged binary content held by PubRouter on behalf of the publisher (see the next section)
-2. A proxy-URL (proxying through the Router) for public content hosted on the web by the publisher
+You can issue a GET request on the URL contained within a link element to receive the content.
 
-In either case you can issue a GET request on the URL to receive the content.
-
-In order to tell the difference between (1) and (2), compare the following two links:
+Links will look like the following and give you the URL and tell you which type of packaging has been used to package up the binary: 
```

Comment:
With records harvested from EPMC we may have external links to FullText on a publisher's  system.  In that case, we still need to describe what the links oject will look like because I think it won't have `packaging` and `type` won't be "package") (which is slightly different from the original text which mentioned proxy).  

(That is, unless the fulltext is downloaded and packaged as part of the Routing process - but it doesn't look to me as if it is).

Here is code from QueryEngineEPMC.py line 187.
```
            notification = IncomingNotification()
            # Add event as acceptance for the Router incoming notification model
            # Add links to the Router incoming notification model
            for link in dictionary_get(document, "fullTextUrlList", "fullTextUrl", default=[]):
                obj = {"url": notification._coerce(link['url'], to_unicode()),
                       "type": _type.get(link['documentStyle'], u'fulltext'),
                       "format": _format.get(link['documentStyle'], u'text/html')}
                notification._add_to_list("links", obj, unique=True)
```